### PR TITLE
#MAN-901 add permalink field for each entity

### PR DESCRIPTION
**Pull request by cdoyle-temple**
Added permalink field to dashboard show pages, using ID data to create url.

### CODE REVIEW

File: app/views/admin/application/show.html.erb [hunk 1]
```diff
@@ -45,14 +45,26 @@ as well as a link to its edit page.
   <dl>
     <% page.attributes.each do |attribute| %>
       <dt class="attribute-label" id="<%= attribute.name %>">
-      <%= t(
-        "helpers.label.#{resource_name}.#{attribute.name}",
-        default: attribute.name.titleize,
-      ) %>
+        <%# binding.pry %>
```
Comment (nomadicoder) [inline app/views/admin/application/show.html.erb:48]: Do we need to keep this around?

Comment (cdoyle-temple) [reply app/views/admin/application/show.html.erb:48]: nope

File: spec/features/dashboard/finding_aid_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  after(:all) do
+    Account.destroy_all
+    FindingAid.destroy_all
+  end
+
+  context "FindingAid SHOW Administrate Page" do
+    scenario "Display existing item show page" do
+      login_as(@admin, scope: :account)
+      visit("/admin/finding_aids/#{@finding_aid.id}")
+      # binding.pry
```
Comment (nomadicoder) [inline spec/features/dashboard/finding_aid_show_spec.rb:20]: Here's one more. Please remove

Comment (cdoyle-temple) [reply spec/features/dashboard/finding_aid_show_spec.rb:20]: argh

**Pull request merged**